code/ui/InventorySelection.cs

```diff
@@ -105,42 +109,63 @@ public override void Tick()
         [Event.BuildInput]
         private void ProcessClientInventorySelectionInput(InputBuilder input)
         {
-            Inventory inventory = Local.Pawn.Inventory as Inventory;
-
-            if (inventory.Count() == 0)
+            if (_children == null || _children.Count == 0)
```

Comment:
Why using `_children` instead of `Children`? Especially regarding upcoming changes of accessibility,